### ISSUE TITLE
Consider a less restrictive license

### ISSUE THREAD

**Issue by keeferrourke** [OpenGenus/cosmos#119]
By licensing this repository under the GNU General Public License, your "library of knowledge" which contains code that is general purpose in nature, is legally restricted to only be used in projects which are GPLv3 compatible.

This is bad for accessibility reasons, and very likely defeats your intended purpose.

Please consider re-licensing under the GNU Lesser Public License ([LGPLv3](https://en.wikipedia.org/wiki/GNU_Lesser_General_Public_License)) or a simpler license such as the [ISC License](https://en.wikipedia.org/wiki/ISC_license) which isn't bogged down by quite so much legalese.

**Comment by AdiChat** [comment]
Thanks for pointing it out. 
We will consider another license, as we move on, which is compatible with the sister projects of this work.

**Comment by LesnyRumcajs** [comment]
Given the number of contributors and range of topics, I'd suggest WTFPL.
https://en.wikipedia.org/wiki/WTFPL

**Comment by keeferrourke** [comment]
The WTFPL is a great license :joy: but it was started as satire and is rather unprofessional. Its legal holding has been challenged in the past, and I only reserve its usage for joke projects. The ISC License is functionally equivalent.

**Comment by bmintz** [comment]
@keeferrourke what's your opinion on MIT? How does it differ?

**Comment by keeferrourke** [comment]
MIT is mostly equivalent to ISC, however the ISC licence has more concise wording.

In nearly all use-cases, I prefer the ISC license over the MIT.
